doc/Learn/Maui/ThirdParty-EsriMaps.md

```diff
@@ -9,54 +9,96 @@ The MapView control, that's part of the ArcGIS Maps SDK for .NET, can be used in
+### [Visual Studio](#tab/vs)
 
-    ```
-    dotnet new unoapp -preset blank -maui -o MauiEmbeddingApp
-    ```
+> [!NOTE]
+> If you don't have the **Uno Platform Extension for Visual Studio** installed, follow [these instructions](xref:Uno.GetStarted.vs2022).
 
-1. Next, add a reference to the [Esri.ArcGISRuntime.Maui NuGet package](https://www.nuget.org/packages/Esri.ArcGISRuntime.Maui) to the `MauiEmbeddingApp.MauiControls` project.
+- Launch **Visual Studio** and click on **Create new project** on the Start Window. Alternatively, if you're already in Visual Studio, click **New, Project** from the **File** menu.
 
-1. In the `AppBuilderExtensions` class, on `MauiEmbeddingApp.MauiControls` project, update the `UseMauiControls` extension method to call the `UseArcGISRuntime` method.
+- Type `Uno Platform` in the search box
 
-    ```cs
-    using Esri.ArcGISRuntime;
-    using Esri.ArcGISRuntime.Maui;
-    using Esri.ArcGISRuntime.Security;
+- Click **Uno Platform App**, then **Next**
 
-    namespace MauiEmbeddingApp;
+- Name the project `ArcGisApp` and click **Create**
 
-    public static class AppBuilderExtensions
-    {
-        public static MauiAppBuilder UseMauiControls(this MauiAppBuilder builder) 
-            => builder
-                .UseArcGISRuntime(
-                //config => config
-                //    .UseLicense("[Your ArcGIS Maps SDK License key]")
-                //    .UseApiKey("[Your ArcGIS location services API Key]")
-                //    .ConfigureAuthentication(auth => auth
-                //        .UseDefaultChallengeHandler() // Use the default authentication dialog
-                //    )
-                )
-                .ConfigureFonts(fonts =>
-                {
-                    fonts.AddFont("MauiEmbeddingApp/Assets/Fonts/OpenSansRegular.ttf", "OpenSansRegular");
-                    fonts.AddFont("MauiEmbeddingApp/Assets/Fonts/OpenSansSemibold.ttf", "OpenSansSemibold");
-                });
-    }
-    ```
+At this point you'll enter the **Uno Platform Template Wizard**, giving you options to customize the generated application.
+
+- Select **Blank** in **Presets** selection
+
+- Select the **Platforms** tab and unselect **Desktop** and **Web Assembly** platforms
+
+- Select the **Features** tab and click on **.NET MAUI Embedding**
 
-    > [!NOTE]
-    > If you have a license key and/or a location service API key, uncomment the `delegate` provided on `UseArcGISRuntime` method. This isn't required to run this sample.
+For more information on all the template options, see [Using the Uno Platform Template](xref:Uno.GettingStarted.UsingWizard).
+
+- Click **Create** to complete the wizard
+
+The template will create a solution with a single cross-platform project, named `ArcGisApp`, ready to run.
```

Comment:
Not too relevant but maybe it would make more sense to follow this order?

```suggestion
- Click **Create** to complete the wizard

The template will create a solution with a single cross-platform project, named `ArcGisApp`, ready to run.

For more information on all the template options, see [Using the Uno Platform Template](xref:Uno.GettingStarted.UsingWizard).
```